packages/react-next/src/Image.ts

```diff
@@ -1,2 +1 @@
-import './version';
-export * from './components/Image/index';
+export * from '@fluentui/react-internal/lib/Image';
```

Comment:
Generally we want these to re-export from react (which re-exports react-internal)
```suggestion
export * from '@fluentui/react/lib/Image';
```